### PR TITLE
About page

### PR DESCRIPTION
**Pull request by BasLee**

### CODE REVIEW

File: examples/development/docker-compose.env [hunk 1]
```diff
@@ -64,12 +75,23 @@ export TR_SWAGGER_SCHEMES=['http']
...
+
+export FULL_TEXT_INDEX=full-text
+export CUSTOM_INDEX=docere
+export AUTOCOMPLETE_INDEX=autocomplete
+export FILE_INDEX=file
 
 export NGINX_LOCATION=/textrepo/
-export NGINX_VIRTUAL_HOST=demorepo.tt.di.huc.knaw.nl
+export NGINX_VIRTUAL_HOST=localhost:8080
+export NGINX_VERTUAL_HOST_SCHEME=http://
```
Comment (hayco) [inline examples/development/docker-compose.env:93]: This introduces a typo: VERTUAL

File: examples/development/docker-compose-dev.yml [hunk 2]
```diff
@@ -156,6 +177,28 @@ services:
...
+        DOCKER_TAG: ${DOCKER_TAG}
+    networks:
+      - textrepo_network
+    volumes:
+      - aboutdata:/about
+    environment:
+      ABOUT_TR_DESCRIPTION: ${ABOUT_TR_DESCRIPTION}
+      ABOUT_TR_DOCUMENTATION: ${ABOUT_TR_DOCUMENTATION}
+      ABOUT_TR_SOURCECODE: ${ABOUT_TR_SOURCECODE}
+      NGINX_VERTUAL_HOST_SCHEME: ${NGINX_VERTUAL_HOST_SCHEME}
```
Comment (hayco) [inline examples/development/docker-compose-dev.yml:199]: Same typo as in docker-compose.env, which is why it still works :-)

File: examples/production/docker-compose.env [hunk 3]
```diff
@@ -66,10 +76,21 @@ export TR_PAGINATION_DEFAULT_LIMIT=10
...
+
+export FULL_TEXT_INDEX=full-text
+export CUSTOM_INDEX=docere
+export AUTOCOMPLETE_INDEX=autocomplete
+export FILE_INDEX=file
+
 export NGINX_LOCATION=/textrepo/
-export NGINX_VIRTUAL_HOST=demorepo.tt.di.huc.knaw.nl
+export NGINX_VIRTUAL_HOST=localhost:8080
+export NGINX_VERTUAL_HOST_SCHEME=http://
```
Comment (hayco) [inline examples/production/docker-compose.env:92]: typo

File: examples/production/docker-compose-integration.yml [hunk 4]
```diff
@@ -145,6 +161,22 @@ services:
...
+    image: knawhuc/textrepo-about:${DOCKER_TAG}
+    networks:
+      - textrepo_network
+    volumes:
+      - aboutdata:/about
+    environment:
+      ABOUT_TR_DESCRIPTION: ${ABOUT_TR_DESCRIPTION}
+      ABOUT_TR_DOCUMENTATION: ${ABOUT_TR_DOCUMENTATION}
+      ABOUT_TR_SOURCECODE: ${ABOUT_TR_SOURCECODE}
+      NGINX_VERTUAL_HOST_SCHEME: ${NGINX_VERTUAL_HOST_SCHEME}
```
Comment (hayco) [inline examples/production/docker-compose-integration.yml:177]: typo

File: examples/production/docker-compose-prod.yml [hunk 5]
```diff
@@ -118,6 +134,21 @@ services:
...
+    image: knawhuc/textrepo-about:${DOCKER_TAG}
+    networks:
+      - textrepo_network
+    volumes:
+      - aboutdata:/about
+    environment:
+      ABOUT_TR_DESCRIPTION: ${ABOUT_TR_DESCRIPTION}
+      ABOUT_TR_DOCUMENTATION: ${ABOUT_TR_DOCUMENTATION}
+      ABOUT_TR_SOURCECODE: ${ABOUT_TR_SOURCECODE}
+      NGINX_VERTUAL_HOST_SCHEME: ${NGINX_VERTUAL_HOST_SCHEME}
```
Comment (hayco) [inline examples/production/docker-compose-prod.yml:148]: typo

File: examples/development/docker-compose.env (hunk 1 above)
Comment (BasLee) [reply examples/development/docker-compose.env]: 6daab35

File: examples/development/docker-compose-dev.yml (hunk 2 above)
Comment (BasLee) [reply examples/development/docker-compose-dev.yml]: 6daab35

File: examples/production/docker-compose.env (hunk 3 above)
Comment (BasLee) [reply examples/production/docker-compose.env]: 6daab35

File: examples/production/docker-compose-integration.yml (hunk 4 above)
Comment (BasLee) [reply examples/production/docker-compose-integration.yml]: 6daab35

File: examples/production/docker-compose-prod.yml (hunk 5 above)
Comment (BasLee) [reply examples/production/docker-compose-prod.yml]: oeps...
6daab35

**Pull request merged**